libs/create_ml_io.py

```diff
@@ -32,6 +32,7 @@ def write(self):
 
         output_image_dict = {
             "image": self.filename,
+            "verified": True if self.verified else False,
```

Comment:
Looks verbose.

`  "verified": True if self.verified else False,`

can be simplified to `"verified":  self.verified`?

Reply:
Ofc, fixed now